opencti-platform/opencti-front/builder/dev/dev-e2e.js

```diff
@@ -0,0 +1,91 @@
+const express = require("express");
+const { createProxyMiddleware } = require("http-proxy-middleware");
+const { readFileSync } = require("node:fs");
+const path = require("node:path");
+const esbuild = require("esbuild");
+const compression = require("compression");
+const { RelayPlugin } = require("../plugin/esbuild-relay");
+const fsExtra = require("fs-extra");
+
+const basePath = "";
+const clients = [];
+const buildPath = "./builder/dev/build/";
+
+const middleware = (target, ws = false) => createProxyMiddleware(basePath + target, {
```

Comment:
No need for a new specific file.
I think you can directly use the dev build.
Chokidar will not be a problem you can keep it.
And the BACK_END_URL env can be added to the dev config file without any impact.

Reply:
In the dev esbuild setup, we force the browser to not cache any content for the live reload by adding no-cache to the headers. 
It might be better to remove this part so the worker can avoid loading the same resources each time for the e2e. 
What do you think?